editor/export/project_export.cpp

```diff
@@ -1395,12 +1395,14 @@ ProjectExportDialog::ProjectExportDialog() {
 
 	set_cancel_button_text(TTR("Close"));
 	set_ok_button_text(TTR("Export PCK/ZIP..."));
+	get_ok_button()->set_tooltip_text("Export all the project files as a PCK or ZIP package.");
```

Comment:
Might be getting a bit verbose, but this might help address @JUSTCAMH's suggestion. And softens the "all the project files" as one might expect this means every single file is exported, while it only exports what Godot considers a resource (or what's configured in the Resources tab).
```suggestion
	get_ok_button()->set_tooltip_text("Export the project resources as a PCK or ZIP package. This is not a playable build, only the project data without a Godot executable.");
```